data/mods/DinoMod/mapgen/zoo.json

```diff
@@ -197,11 +197,7 @@
       },
       "item": {
         "I": { "item": "id_science_transport_1", "chance": 15 },
-        "t": [
-          { "item": "plunger_toilet", "prob": 90 },
-          { "item": "brush_toilet", "prob": 75 },
-          { "item": "toilet_paper", "prob": 95 }
-        ]
+        "t": [ { "item": "plunger_toilet" }, { "item": "brush_toilet" }, { "item": "toilet_paper" } ]
```

Comment:
This line wants removing bc the SUS_toilet itemgroup below already adds these

Reply:
Item spawn changes can be done in a separate subsequent PR, this PR avoids modifying existing mapgen behavior.

Reply:
I thought this is what the bot pinged LyleSY for?

Reply:
The pings are for peeps who've opted in to getting pinged for specific files in folders changes usually so modders are aware someone is changing their mod and can have a final say/propose alterations to fit the mod etc, not with the idea that they have to do every little thing related to their mod (also LyleSY is inactive)

Reply:
I mean ye and the other bits where you're removing sensical (ie like not the ammo 100 magazine 100 ones) but non-functional fields they either want fixing (making item groups for the most part) or at least leaving in commented out